src/Raven.Server/Commercial/LicenseType.cs

```diff
@@ -8,7 +8,7 @@ public enum LicenseType
 
         Community = 1,
 
-        Trial = 2,
+        Essential = 2,
```

Comment:
I don't think this is correct. 
We don't want to _replace_ `Trial` (although that isn't used at all for us).
We need a new value (and changing `Trial => Reserved`)

Another thing to consider is the backward compatibility here, no?

Reply:
done